charts/dynatrace-oneagent-operator/v0.8.1/questions.yml

```diff
@@ -27,7 +27,7 @@ questions:
   description: "Your Dynatrace API token - You can generate this token in your Dynatrace environment"
   default: ""
   type: string
-  required: true
+  required: false
   group: "Agent Configuration (REQUIRED)"
```

Comment:
Nit pick: Can you remove all the "(REQUIRED)" from this group? Not all the values in it have the `required: true` so is a bit confusing. Also, the UI displays a red mark next to entries with the `required: true` and won't let you move forward if left blank.

Reply:
Done!